examples/video_mode/test_PID.py

```diff
@@ -0,0 +1,186 @@
+from matplotlib import pyplot as plt
```

Comment:
names starting with `test_` are commonly used for testing with PyTests. Let's rename to PID_example? or just PID?

same for user input